621.py

```diff
@@ -0,0 +1,20 @@
+class Solution:
+    def leastInterval(self, tasks: List[str], n: int) -> int:
+        task_freqs = Counter(tasks)
+        sorted_freqs = sorted(task_freqs.values(), reverse=True)
```

Comment:
The code between line 4 and line 12 is too verbose and doesn't have a good performance due to the sorting part. I would write it this way:

```python
max_freq = max(task_freq.values())
max_count = task_freq.values().count(max_freq)
```